### DISCUSSION TITLE
Logistics Standalone?

### DISCUSSION THREAD

**Discussion by Uncreatedlemon91** [Vdauphin/HeartsAndMinds#1010] [Question and answer]
Is the logistics system (FOB Placement, Wrecks being repaired, Logistic Creation) able to be used standalone? It's a very robust system and I'm curious if it's possible to use that with an ALiVE scenario? Obviously credit would be given as needed!

**Comment by canprod** [comment]
mdr

**Comment by Vdauphin** [comment]
Hello,

> Is the logistics system (FOB Placement, Wrecks being repaired, Logistic Creation) able to be used standalone?

Sadly, this mission isn't modular as ACE is for example. All feature are in its own folder but there is a lot of function call between feature. So, it would be hard to make it standalone.
Also, there is plenty of great standalone stuff. Most feature in H&M are only useful in this mission.

>  It's a very robust system and I'm curious if it's possible to use that with an ALiVE scenario? 

If you know what you are doing and have time, it is always possible.
May be a start would be to disable city detection by removing https://github.com/Vdauphin/HeartsAndMinds/blob/master/%3DBTC%3Dco%4030_Hearts_and_Minds.Altis/core/init_server.sqf#L1
and
https://github.com/Vdauphin/HeartsAndMinds/blob/master/%3DBTC%3Dco%4030_Hearts_and_Minds.Altis/core/init_server.sqf#L14-L15
as I think you want to use ALIVE for units creation.

Cheers.

**Reply by Uncreatedlemon91** [reply]
Fantastic, I'll have a tinker and see what I can come up with....appreciate your dedication and work on this mission!

**Reply by Vdauphin** [reply]
Also, welcome back and good luck!
